terraform/variables.tf

```diff
@@ -103,6 +103,27 @@ variable "redis_cache_size" {
   description = "Size of the Redis instance in GB."
 }
 
+variable "adminapi_custom_domain" {
+  type    = string
+  default = ""
+
+  description = "Custom domain to map for adminapi. This domain must already be verified by Google. If not provided, no domain mapping is created."
```

Comment:
We should also state that a DNS CNAME record pointing to `ghs.googlehosted.com.` should be created in advance.